spec/index.bs

```diff
@@ -2821,19 +2826,16 @@ enum GPUQueryType {
 ## Pipeline Statistics Query ## {#pipeline-statistics}
 
 <script type=idl>
-typedef [EnforceRange] unsigned long GPUPipelineStatisticFlags;
-interface GPUPipelineStatisticBit {
-    const GPUPipelineStatisticFlags VERTEX_SHADER_INVOCATIONS   = 0x01;
-    const GPUPipelineStatisticFlags CLIPPER_INVOCATIONS         = 0x02;
-    const GPUPipelineStatisticFlags CLIPPER_PRIMITIVES_OUT      = 0x04;
-    const GPUPipelineStatisticFlags FRAGMENT_SHADER_INVOCATIONS = 0x08;
-    const GPUPipelineStatisticFlags COMPUTE_SHADER_INVOCATIONS  = 0x10;
+enum GPUPipelineStatisticName {
+    "vertex_shader_invocations",
+    "clipper_invocations",
+    "clipper_primitives_out",
+    "fragment_shader_invocations",
+    "compute_shader_invocations"
 };
 </script>
 
-When resolving the results of a pipeline statistics query, the number of pipeline statistics written into GPU buffer depends on the number of enabled {{GPUPipelineStatisticBit}} in {{GPUQuerySetDescriptor/pipelineStatistics}}.
-
-Each result is written into unsigned 64-bit integer and in the order of the lowest-valued member of the {{GPUPipelineStatisticBit}} to the highest.
+When resolving the results of a pipeline statistics query, each result is written into unsigned 64-bit integer, and the number and order of pipeline statistics written into GPU buffer depends on the number and order of {{GPUPipelineStatisticName}} specified in {{GPUQuerySetDescriptor/pipelineStatistics}}.
```

Comment:
Here we can be stronger and say that the order and number of the u64 written matches the order of pipelineStatistics. (instead of "depends" without saying how it depends from it)